utils/split_by_subset.f90

```diff
@@ -25,15 +25,12 @@ program split_by_subset
 
   !> get input filename from stdin, if not use 'fort.20'
   call getarg(1, finput)
-  if (trim(adjustl(finput)) == '') finput = 'fort.20'
-
-  !> check if file exists, then open it, else abort
   inquire(file=trim(adjustl(finput)), exist=file_exists)
   if (file_exists) then
     open(lunit, file=trim(adjustl(finput)), form='unformatted')
     call openbf(lunit,'IN',lunit)
   else
-    call bort('File ' // trim(adjustl(finput)) // ' does not exist')
+    call bort('Usage: "split_by_subset bufrfile" will split a BUFR file into subsets')
```

Comment:
Maybe I'm missing something here, but it seems like the old error message was more appropriate.  The new error message doesn't seem appropriate for when the boolean "file_exists" tests false.

Reply:
Only remaining comment/concern I have is that the old error message still seems more appropriate for this "if" test.

Looking at some of the other utility codes which have similar logic (e.g. cmpbqm, gettab, readmp), those all do a prior separate "if (file == '')" test with their own bort message similar to the usage diagnostic you now have, followed by a second separate "if" test on the result of the inquire with a bort message similar to the old error message.  So why is this utility any different?